src/schemas/team.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Team",
+  "type": "object",
+  "properties": {
+    "owner": {
+      "type": "string"
+    },
+    "players": {
+      "type": "array",
+      "items": {
+        "$ref": "./player.json"
+      }
```

Comment:
Might make sense to add some scores (probably weekly ones) here too

Reply:
Maybe just calculate and show scores locally on the UI for now in case scoring multipliers change?